tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
```

Comment:
And for the same project - yup, it's definitely a good feature that it could be used independently. It can still be considered as part of the project, though, it's not exclusive : )